boxsdk/object/base_endpoint.py

```diff
@@ -17,7 +17,8 @@ def __init__(self, session):
         """
         self._session = session
 
-    def get_url(self, endpoint, *args):
+    @classmethod
```

Comment:
I don't think it's a good idea to make this a classmethod when subclasses will not make it a classmethod.